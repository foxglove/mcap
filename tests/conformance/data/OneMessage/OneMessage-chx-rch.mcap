version https://git-lfs.github.com/spec/v1
oid sha256:2b7f9280ec905e67965b872c60d02433a9a7968ae1f2acfc52b4346d477aae00
size 235
