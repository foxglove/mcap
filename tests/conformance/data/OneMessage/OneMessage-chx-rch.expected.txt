Header library="" profile=""
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Footer summary_crc=0 summary_offset_start=0 summary_start=151
